ts/src/main/scala/ch/epfl/ts/evaluation/EvaluationRunner.scala

```diff
@@ -24,42 +27,60 @@ object EvaluationRunner {
     // Fetcher
     // variables for the fetcher
     val dateFormat = new java.text.SimpleDateFormat("yyyyMM")
-    val startDate = dateFormat.parse("201503");
-    val endDate   = dateFormat.parse("201503");
-    val workingDir = "./data";
-    val currencyPair = "EURCHF";
-    val fetcher = builder.createRef(Props(classOf[HistDataCSVFetcher], workingDir, currencyPair, startDate, endDate, 60.0),"HistFetcher")
+    val startDate = dateFormat.parse("201301");
+    val endDate = dateFormat.parse("201312");
+    val workingDir = "/Users/admin";
```

Comment:
Avoid committing a path specific to your computer to a common repo.
If needed, create a symlink from your data directory to `./data`